src/components/HathorHeader.js

```diff
@@ -76,12 +75,11 @@ const HathorHeader = (props) => {
 
 const styles = StyleSheet.create({
   wrapper: {
-    height: HEADER_HEIGHT,
+    height: STYLE.headerHeight,
     flexDirection: 'row',
-    alignItems: 'center',
-    borderColor: '#eee',
+    alignItems: 'flex-end',
+    borderColor: COLORS.borderColor,
     paddingHorizontal: 16,
-    backgroundColor: baseStyle.title.backgroundColor,
```

Comment:
Why did you remove this?

Reply:
A [discussion on slack](https://hathornetwork.slack.com/archives/G01EXLHAZUP/p1694008418391819?thread_ts=1694008353.176049&cid=G01EXLHAZUP) made me notice this background color was not consistent with what was implemented on previous versions ( 0.25.0 and before ).

This removal fixed it, as now the Header background color is the same as the rest of the main application colors. See `Dashboard` and `Settings` screenshots on the PR description for the result of this change.